src/scss/components/checkbox/mixins/_hxCheckboxControl.scss

```diff
@@ -1,28 +1,33 @@
-//
-// ===== Mixins for <hx-checkbox-control> states =====
-//
+// =================================================== //
+// ===== Mixins for <hx-checkbox-control> states ===== //
+// =================================================== //
 
 // ========== PRISTINE ==========
 @mixin __checkboxControl($pseudo-state: null) {
   @if $pseudo-state == focus {
-    box-shadow: $focus-glow;
+    // box-shadow: $focus-glow; // TBD v2.0 specs
```

Comment:
So has designed confirmed that we are just getting rid of the box-shadow for all focus states on every element?
If we have them on some elements, but not others, then that is inconsistent. Let me know if they have a made a decision.